docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device/performance-hint-and-threads-scheduling.rst

```diff
@@ -0,0 +1,163 @@
++======================================+================================================================+================================================================+
+| ``ov::num_streams``                  | 1                                                              | 1                                                              |
++--------------------------------------+----------------------------------------------------------------+----------------------------------------------------------------+
+| ``ov::inference_num_threads``        | is equal to number of P_cores or P_cores+E_cores on one socket | is equal to number of P_cores or P_cores+E_cores on one socket |
++--------------------------------------+----------------------------------------------------------------+----------------------------------------------------------------+
+| ``ov::hint::scheduling_core_type``   | `Core Type Table of Latency Hint`_                             | `Core Type Table of Latency Hint`_                             |
++--------------------------------------+----------------------------------------------------------------+----------------------------------------------------------------+
+| ``ov::hint::enable_hyper_threading`` | No                                                             | No                                                             |
++--------------------------------------+----------------------------------------------------------------+----------------------------------------------------------------+
+| ``ov::hint::enable_cpu_pinning``     | No / Not Support                                               | Yes except using P-cores and E-cores together                  |
++--------------------------------------+----------------------------------------------------------------+----------------------------------------------------------------+
+
+.. note::
+
+    - ``ov::hint::scheduling_core_type`` might be adjusted for particular inferred model on particular platform based on internal heuristics to guarantee best performance.
+    - Both P-cores and E-cores are used for the Latency Hint on Intel® Core™ Ultra Processors on Windows, except in the case of large language models.
+    - In hyper threading situation, two logical processors share hardware resource of one CPU core. OpenVINO do not expect to use both logical processors in one stream for one infer request. So ``ov::hint::enable_hyper_threading`` is ``No`` in this scenario.
+    - Usually user on Windows will run multiple applications at the same time, so ``ov::hint::enable_cpu_pinning`` for Windows is ``No`` on single socket platform and ``Not Supported`` on dual sockets platform. However user on Linux request the best performance of one application, so ``ov::hint::enable_cpu_pinning`` is ``Yes`` for Linux.
```

Comment:
Statements like "Usually user on Windows" and "However user on Linux request the best performance of one application" sounds too bold from my pespective. I would rephrase: 
"``ov::hint::enable_cpu_pinning`` is disabled by default on Windows/Mac, and enabled on Linux". Such default settings are aligned with typical workloads running in corresponding environment to guarantee better OOB performance.